cores/esp8266/MD5Builder.cpp

```diff
@@ -1,101 +1,19 @@
 #include <Arduino.h>
```

Comment:
Can't comment on them directly, but there is a bunch of binary files with .gch extensions added in this PR.